manifests/x/x64dbg/x64dbg/snapshot_2023-10-05_13-38/x64dbg.x64dbg.installer.yaml

```
@@ -0,0 +1,25 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: x64dbg.x64dbg
+PackageVersion: snapshot_2023-10-05_13-38

```

```suggestion
PackageVersion: 2023.10.05
```
The version should be either a semantic version or a date in order for Winget to sort the newest releases properly